### PR TITLE
Feature/san 6599

### PR DESCRIPTION
**Pull request by GarageInc**
**Summary**
1. Show shared signal for anonymouse users https://favro.com/organization/bdbb4f24afc48b29c74f4fa4/cffeb6e3eaf254588cb68a9a?card=San-6599

**Screenshots**
image

### CODE REVIEW

**Comment by Partysun** [comment]
@GarageInc What is happened if anonymous user clicks on 
image

File: src/ducks/Signals/signalFormManager/signalMaster/SignalMaster.js [hunk 1]
```diff
@@ -143,8 +146,12 @@ const SignalMaster = ({
         <SharedTriggerForm
           id={stateTrigger.id}
           trigger={stateTrigger}
-          onOpen={setOpenSharedForm}
-          onCreate={() => handleSettingsChange(settings)}
+          onOpen={data =>
+            isLoggedIn ? setOpenSharedForm(data) : setAnonWarning(true)
```
Comment (Partysun) [inline src/ducks/Signals/signalFormManager/signalMaster/SignalMaster.js]: setAnonWarning can have default value. Did you pass false in other places?
From naming of this func it should works without arguments. @GarageInc

File: src/ducks/Signals/signalModal/SignalMasterModalForm.js [hunk 2]
```diff
@@ -164,8 +164,17 @@ const MainDialog = ({
   buttonParams
 }) => {
   const [dialogTitle, onSetDialogTitle] = useState('')
+  const [isAnonWarning, setAnonWarning] = useState(false)
```
Comment (Partysun) [inline src/ducks/Signals/signalModal/SignalMasterModalForm.js]: ah., this is a hook

File: src/ducks/Signals/signalFormManager/signalMaster/SignalMaster.js (hunk 1 above)
Comment (GarageInc) [reply src/ducks/Signals/signalFormManager/signalMaster/SignalMaster.js]: It is good point. Child should't know about props of parent.

**Comment by GarageInc** [comment]
> @GarageInc What is happened if anonymous user clicks on
> image

He will see a small popup:
image

**Pull request merged**